core/scripts/ocr2vrf/setup_ocr2vrf.go

```diff
@@ -179,4 +179,24 @@ func setupOCR2VRFNodes(e helpers.Environment) {
 
 	fmt.Println()
 	fmt.Println("Consumer address:", consumerAddress.String())
+	fmt.Println("Consumer request command:")
```

Comment:
If we're adding extra to the setup in this PR i will say it would be nice to print the coordinator/DKG/Consumer addresses explicitly at the end of the setup. It's annoying going back to the beginning or searching the commands for them.

Reply:
They're printed as part of the respective `set-config` commands, but I can add a separate log line to print the address of each more clearly.

Reply:
nvm recent runthrough I see you've taken the liberty of adding that log.

Reply:
Previously the consumer address was missing and I had to scroll a bit to find it.